### PR TITLE
Tyk cloud quick start and overview improvements

### PR DESCRIPTION
**Pull request by SonjaChevre**
### For internal users - please add a Jira DX PR ticket to the subject!
---
### Description

@letzya: the new quick start for Tyk Cloud

Tyk Cloud overview: https://deploy-preview-2963--tyk-docs.netlify.app/docs/nightly/tyk-cloud/
What is Tyk Cloud: https://deploy-preview-2963--tyk-docs.netlify.app/docs/nightly/tyk-cloud/what-is-tyk-cloud/
New Quickstart: https://deploy-preview-2963--tyk-docs.netlify.app/docs/nightly/deployment-and-operations/tyk-cloud-platform/quick-start/

@letzya: can we maybe hide the old quick start (https://deploy-preview-2963--tyk-docs.netlify.app/docs/nightly/tyk-cloud/getting-started/) in the navigation? we might still need it as e-mails sent from HubSpot link to those pages (and maybe other pages). But I would rather re-use this content to rewrite and improve some other pages. And get rid of a quick start that is 7 pages long.

@mitjaziv: since I have was working on the doc, I made the change mentioned in https://tyktech.atlassian.net/browse/DX-527
also happy for you to review and provide feedback!

## Checklist

- [x ] I have added a **preview link** to the PR description.
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/content/deployment-and-operations/tyk-cloud-platform/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+menu:
+  main:
+    parent: "Tyk Cloud"
+---
+
+Start using Tyk on our servers. Nothing to install. 
+
+## Step 1: Sign Up for Tyk Cloud
+
+To beginn your Tyk Cloud journey, follow these simple steps to sign up for an account:
```
Comment (dcs3spp) [inline tyk-docs/content/deployment-and-operations/tyk-cloud-platform/quick-start.md]:
```suggestion
To begin your Tyk Cloud journey, follow these simple steps to sign up for an account:
```

File: tyk-docs/content/deployment-and-operations/tyk-cloud-platform/quick-start.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+* Navigate to the [Tyk Cloud sign up form](https://tyk.io/sign-up/#cloud).
+* Fill in the required information and click on "Next step: Create your password."
+* Provide your email address and choose a robust password for your account.
+* Select your home region, where your data will be securely stored.
+* Opt for the "Set up API platform automatically" option (You can still personalize your setup later).
```
Comment (dcs3spp) [inline tyk-docs/content/deployment-and-operations/tyk-cloud-platform/quick-start.md]:
```suggestion
* Opt for the "Set up API platform automatically" option (you can still personalize your setup later).
```

File: tyk-docs/content/deployment-and-operations/tyk-cloud-platform/quick-start.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+* Wait a couple of minutes and congratulations, your API platform was deployed!
+
+By default, a cloud data plane will be deployed for you. You can also deploy hybrid data planes on your own infrastructure. 
+
+## Step 2: Get started with your first API with Tyk Dashboard
```
Comment (dcs3spp) [inline tyk-docs/content/deployment-and-operations/tyk-cloud-platform/quick-start.md:27]: Steps look good to me. For this section maybe add screenshot to accompany the steps

File: tyk-docs/content/deployment-and-operations/tyk-cloud-platform/quick-start.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+* Keep https://httpbin.org/ as the upstream URL and click on "Configure API"
+* Select to which gateway you want to deploy this API, select the "edge" tags to deploy to the cloud data plane.
+* Customize your API settings, including authentication, rate limits, and caching, as per your requirements.
+* Click "Save" to create your API. Congratulations! You've just set up your first API.
+
+Now that your API is created, you can explore and manage it through the Tyk Dashboard.
+
+## Next Steps
+
+This quick start guide has covered the essentials to get you up and running with Tyk Cloud. As you explore further, you might want to learn more about managing APIs in Tyk Dashboard or managing the infrastructure and environments in Tyk Cloud.
```
Comment (dcs3spp) [inline tyk-docs/content/deployment-and-operations/tyk-cloud-platform/quick-start.md]: Add links for managing APIs and managing infrastructure and environments

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Hi @SonjaChevre This looks great and reads well. I have had a look through and made minor suggestions

File: tyk-docs/content/deployment-and-operations/tyk-cloud-platform/quick-start.md (hunk 4 above)
Comment (SonjaChevre) [reply tyk-docs/content/deployment-and-operations/tyk-cloud-platform/quick-start.md]: will add links tomorrow morning and do a last check! thanks for your review.

Comment (SonjaChevre) [reply tyk-docs/content/deployment-and-operations/tyk-cloud-platform/quick-start.md]: added all relevant links

**Comment by SonjaChevre** [comment]
All good from my side, @dcs3spp please review the latest version.

**Review by dcs3spp — APPROVED** [review]
That's great thanks Sonja!!! Approved

File: tyk-docs/data/menu.yaml [hunk 5]
```diff
@@ -98,11 +98,11 @@ menu:
           show: True
         - title: "What Is Tyk Cloud?"
           path: /tyk-cloud/what-is-tyk-cloud
+          show: False
```
Comment (letzya) [inline tyk-docs/data/menu.yaml:101]: should have been true

**Pull request merged**